src/jsutils.h

```diff
@@ -57,21 +60,30 @@ size_t flash_strlen(const char *str);
 /// Copy a string from flash to RAM
 char *flash_strncpy(char *dest, const char *source, size_t cap);
 
+/// Compare a string in memory with a string in flash
+int flash_strcmp(const char *mem, const char *flash);
```

Comment:
agreed - however this gets confusing reading the code for a board which it does not apply - it *looks* like it's using `flash_strncmp` - but isn't!

would something like `safe_strcmp` be more neutral?